tools/build_file_generator/JavaClassAnalyzer.java

```diff
@@ -114,6 +233,53 @@ private static Import getImport(String importLine) {
     return result.build();
   }
 
+  private List<String> getUsedClassnamesInCode(String fileContent) {
+    final String multilineCommentRegex = "/\\*(?:.|[\\n\\r])*?\\*/";
```

Comment:
Please document what these regexes do (or rather, how they do it).

Reply:
Done.